procrastinate/exceptions.py

```diff
@@ -63,6 +63,12 @@ def __init__(
         self.critical = critical
 
 
+class JobAborted(ProcrastinateException):
+    """
+    Job was aborted (usually as reaction to an abortion request).
```

Comment:
This message will end up in the logs unless users put a special message. It's less documentation than it's communication. I think `Job was aborted` should be good.